### PR TITLE
Feat: Adjustment of the video on the homepage

### LINKED ISSUE #63: Adjustment of the video on the homepage
The video has to be stretched out to fill out the screen. Right now it jumps to a small square halfway through. 
YouTube elements, such as title, photo of the owner of the video, share button and so on for example, have to be invisible.

### PR DESCRIPTION
**Pull request by dkostenko-bjerk**
- Adjusted video on the home page
- Added mp4 file to the assets and its importing (removed Youtube-embedding with it's title)

Close #63

### CODE REVIEW

File: web/src/pages/index.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /** @jsx */
 import { Box, Grid, Heading, Label } from '@theme-ui/components';
 import React from 'react';
+import BjerkVideo from '../assets/Bjerk.mp4';
```
Comment (cobraz) [inline web/src/pages/index.tsx:4]: This doesn't play on my computer (MacOSX, Chrome). However, isn't it better to use WEBM?

**Review by cobraz — CHANGES_REQUESTED** [review]

File: web/src/pages/index.tsx (hunk 1 above)
Comment (dkostenko-bjerk) [reply web/src/pages/index.tsx]: @cobraz Yes, I changed to the .webm

**Review by AnneMatilde — CHANGES_REQUESTED** [review]
Please set video height to screen size. Everything else looks good!

**Review by cobraz — APPROVED** [review]

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde for what resolutions (maybe from and to) it should correspond to the size of the window?

**Review by cobraz — APPROVED** [review]

**Comment by cobraz** [comment]
> Please set video height to screen size. Everything else looks good!

I think you misunderstood this 👆 AFAIK, the video shouldn't be full height of the browser (`100vh` or `100%`). It should have some maximum height; which I think @AnneMatilde can tell you what is 👍

**Comment by AnneMatilde** [comment]
I think that @cobraz actually misunderstood. I do want the video to be full height of the browser window – on all resolutions. I want the video to be centered and to fill the screen size.

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde Now it is, but in mobile design template is not full height

**Comment by AnneMatilde** [comment]
Og right, thats true @dkostenko-bjerk! Please set video height to 70% on mobile.

**Comment by dkostenko-bjerk** [comment]
Ok @AnneMatilde , but in #97

**Comment by AnnaBjerk** [comment]
> * Adjusted video on the home page
> * Added mp4 file to the assets and its importing (removed Youtube-embedding with it's title)
> 
> Close #63

A couple of seconds into the video it turns into a small square.

**Comment by AnneMatilde** [comment]
@AnnaBjerk the video itself turns into a square – thats just how the clip is 😊

**Comment by AnneMatilde** [comment]
It seems like the text is now underneath the video yes @dkostenko-bjerk – but the text on the video does not have the same inner container.
image

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde I think it was fixed in #119

**Comment by AnneMatilde** [comment]
No, its still the same on bigger screens. 
image

The text should be aligned with the logo.

**Review by cobraz — APPROVED** [review]

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde , it's done

**Comment by cobraz** [comment]
image

This part should probably be moved down (so the video shouldn't be underneath «Hvordan kan vi hjelpe deg», that container should move down).

Anne Matilde should probably confirm :D

/cc @AnneMatilde

File: web/src/components/home-page/intro-video.tsx [hunk 2]
```diff
@@ -62,18 +49,16 @@ const IntroVideo: React.FC<{ data: IntroVideoProps }> = ({
...
-          width="100%"
-          height="100%"
           style={{
             position: 'absolute',
-            top: 0,
-            left: 0,
-            width: '100%',
-            height: '100%',
+            inset: '-100%',
+            margin: 'auto',
```
Comment (cobraz) [inline web/src/components/home-page/intro-video.tsx:61]: Maybe you shouldn't use `position` for this?

**Comment by AnneMatilde** [comment]
Yes, correct. The video should "push" the other content down. 
@dkostenko-bjerk

**Pull request closed without merging**